cmd/ocm-backplane/cloud/credentials_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	bpCredentials "github.com/openshift/backplane-cli/pkg/credentials"
```

Comment:
Suggest moving to line 25 to make backplane-cli import in the same group.

Reply:
Good call, done